src/vivarium_public_health/population/base_population.py

```diff
@@ -202,6 +202,31 @@ def _validate_config_for_deprecated_keys(self) -> None:
                 f"Public Health. Use the new key '{mapper[key]}' instead."
             )
 
+    def _load_population_structure(self, builder: Builder) -> pd.DataFrame:
+        return load_population_structure(builder)
+
+
+class ScaledPopulation(BasePopulation):
+    """Component for scaling the population structure based on a scaling factor."""
+
+    def __init__(self, scaling_factor: str | float | pd.DataFrame):
```

Comment:
Can scaling_factor be an int?

Reply:
I think we actually don't want to allow it to be a float or an int, since a scalar scaling factor would have no effect.